docs/source/en/model_doc/madlad-400.md

```diff
@@ -48,7 +48,6 @@ One can directly use MADLAD-400 weights without finetuning the model:
 >>> model = AutoModelForSeq2SeqLM.from_pretrained("google/madlad400-3b-mt")
 >>> tokenizer = AutoTokenizer.from_pretrained("google/madlad400-3b-mt")
 
->>> inputs = tokenizer("<2pt> I love pizza!", return_tensors="pt")
```

Comment:
@cyyever this one as well

(last relevant one I could find :) )

Reply:
Fixed